src/main/java/pl/jkap/sozzt/contract/domain/ContractConfiguration.java

```diff
@@ -1,13 +1,20 @@
 package pl.jkap.sozzt.contract.domain;
 
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 public class ContractConfiguration {
```

Comment:
I don't quite understand this approach, first one looks like a stub and second a proper bean, is this what you wanted to do here?

Reply:
Are there any reason why you are not using spring mechanisms here?